test/test_overrides.py

```diff
@@ -649,7 +649,11 @@ def instance_gen():
                     func_args.append(3.5)
                 elif t == 'bool':
                     func_args.append(False)
-                elif t.startswith('int') or t in {'Dimname', 'DimnameList'}:
+                elif t == 'Dimname':
+                    func_args.append("")
+                elif t == 'DimnameList':
+                    func_args.append([""])
+                elif t.startswith('int'):
                     func_args.append(0)
```

Comment:
@peterbell10 does this change sound reasonable?

Reply:
Seems reasonable to me, a dimname should be a string in the python api.